tests/edgeless/linked-doc.spec.ts

```diff
@@ -42,13 +42,6 @@ test.describe('note to linked doc', () => {
     await waitNextFrame(page, 200);
     const embedSyncedBlock = page.locator('affine-embed-synced-doc-block');
     assertExists(embedSyncedBlock);
-
-    await triggerComponentToolbarAction(page, 'linkedDocInfo');
```

Comment:
If you want to remove these tests, you should add new one to protect your feature.